maintainers/maintainer-list.nix

```diff
@@ -9559,6 +9559,12 @@
     githubId = 782440;
     name = "Luna Nova";
   };
+  lurkki = {
```

Comment:
Add yourself to maintainers list as a separate commit (`maintainers: add lurkki`).